coursefinder/templates/coursefinder/course_finder_results.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+{% load static %}
+{% load wagtailcore_tags %}
+{% block body_class %}template-course-finder-results{% endblock %}
+{% block extra_css %}
+<link rel="stylesheet" href="{% static 'css/coursefinder.css' %}">
+{% endblock extra_css %}
+
+{% block content %}
+<div class="course-finder-base">
+  <div class="header">
+    <h2>Course Finder</h2>
+  </div>
+  <ul class="navigation">
+      <li class="helper-text back">Back</li>
+      <a href={% pageurl page.get_parent %} onclick="handleStartAgain()"><li class="helper-text start-again">Start again</li></a>
+      <li class="helper-text filter">Filter</li>
+  </ul>
+  <div class="course-finder-content">
+     {% include "results_base.html" %}
```

Comment:
Just being pedantic but this is not really a base template if you are using include. Base templates should be used at the top of other templates using the extends tag.

So it might be worth changing the `results_base.html` to something akin to `search_result_list.html`, this explicitly defines the html.

Reply:
Yep I agree, I'll change this.